MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -863,8 +863,13 @@ private MediaStream GetMediaStream(bool isAudio, MediaStreamInfo streamInfo, Med
                     }
                 }
             }
+            else if (string.Equals(streamInfo.CodecType, "data", StringComparison.OrdinalIgnoreCase))
+            {
+                stream.Type = MediaStreamType.Data;
+            }
             else
             {
+                _logger.LogError("Codec Type unknown. The stream will be ignored. Warning: Subsequential streams will have a wrong stream specifier!");
```

Comment:
I would add a parameter to the log to know exactly what the unknown codec type is

Reply:
Done in commit a51cd4f8db6d00496bdf39e93254c25e1baf2ae6